src/FsCodec.Box/TryDeflate.fs

```diff
@@ -9,9 +9,17 @@ module private EncodedMaybeDeflated =
     type Encoding =
         | Direct = 0
         | Deflate = 1
+        | Brotli = 2
```

Comment:
guess module gets a better name on L7 and L73 (compress?)

Reply:
probably `EncodeTryDeflate` and anything with substring `TryDeflate` should become `TryCompress` ?